eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/data/HttpConnectRecord.java

```diff
@@ -46,7 +51,9 @@ public class HttpConnectRecord {
     public static HttpConnectRecord convertConnectRecord(ConnectRecord record, String type) {
         return HttpConnectRecord.builder()
             .type(type)
-            .timestamp(LocalDateTime.now().toString())
+            .time(LocalDateTime.now().toString())
+            .uuid(UUID.randomUUID().toString())
+            .eventId(type + "-" + record.getTimestamp())
             .data(record)
```

Comment:
IMO, timestamp field should be a standalone field. `eventId` can concat a record's offset.

Reply:
Please refer to `org.apache.eventmesh.connector.rocketmq.source.connector.RocketMQSourceConnector#convertToRecordOffset`. The value of this Map is the offset value.

Reply:
Okay, I will make the following changes:
```java
    public static HttpConnectRecord convertConnectRecord(ConnectRecord record, String type) {
        Map<String, ?> map = record.getPosition().getOffset().getOffset();
        String offset = "";
        if (!map.isEmpty()) {
            offset = map.values().iterator().next().toString();
        }
        return HttpConnectRecord.builder()
            .type(type)
            .time(LocalDateTime.now().toString())
            .uuid(UUID.randomUUID().toString())
            .eventId(type + "-" + offset)
            .data(record)
            .build();
    }
```
ps: Is there anything else that needs to be changed?

Reply:
no more~

Reply:
Resolved at https://github.com/apache/eventmesh/pull/4837/commits/3dcc5a978bfc05315de31bb28b799c49c9ff9a54